res/values/arrays.xml

```diff
@@ -84,12 +84,12 @@
       <item>te</item>
   </string-array>
 
-    <string-array name="pref_theme_entries">
+    <string-array name="pref_theme_entries" tools:ignore="InconsistentArrays">
```

Comment:
what does `tools:ignore="InconsistentArrays"` do? and why is it needed for this array but not for the others?

Reply:
In one config, we have only 2 entries, in another config we have 3 entries (because "follow system" was added as an option). That's what Android would complain about without the warning.

Reply:
k, i would also add that bit as a short comment to the code.